strings/crustacean_replacement.json

```diff
@@ -0,0 +1,158 @@
+		"problem": "prawnblem",
+		"proba": "prawnba",
+		"prob": "prawn",
+		"pawn": "prawn",
+		"cool": "tenta-cool",
+		"bat": "bait",
+		"brilliant": "gilliant",
+		"totally": "turtally",
+		"confu" : "conchfu",
+		"kungfu": "conchfu",
+		"kung fu": "conch fu",
+		" mull": " mullet",
+		"believe": "bereef",
+		"believ": "bereef",
+		"rage": "rayge",
+		"kid": "squid",
+		"mark": "shark",
+		"tray": "cray",
+		"cry": "cray",
+		"pray": "cray",
+		"comp": "clam",
+		"calm": "clam",
+		"calam": "clam",
+		"ambulance": "clambulance",
+		"bike": "pike",
+		"suspicious": "fishy",
+		" sus ": " fishy ",
+		"sussy": "sushi",
+		"tune": "tuna",
+		"opportunity": "opportunaty",
+		"got ": "cod ",
+		"god": "cod",
+		"caught": "cod",
+		"grapple": "crapple",
+		"grabb": "crappl",
+		" rue": " roe",
+		"grab": "crapple",
+		"self": "shellf",
+		"cell": "shell",
+		"come": "chum",
+		"plank ": "plankton ",
+		"meant": "manta",
+		"skill": "krill",
+		"chill": "krill",
+		"core": "coral",
+		"hell of it": "halibut",
+		"hell": "eel",
+		"whole": "whale",
+		"muscle": "mussel",
+		"cracking" : "kraken",
+		"crackin": "kraken",
+		"opinion": "octopinion",
+		"utter": "otter",
+		"neme": "nemo",
+		"real": "reel",
+		"orc ": "orca ",
+		"ork ": "orca ",
+		"macaroni": "mackarel",
+		"usurer": "loan shark",
+		"chewy": "spongey",
+		"hamburger": "crabby patty",
+		"burger": "patty",
+		"crust": "krust",
+		"offic": "ofish",
+		"outer space": "trouter space",
+		"deepspace": "trouter space",
+		"deep space": "trouter space",
+		"surgeon": "sturgeon",
+		"purpose": "porpoise",
+		"bastard": "basstard",
+		"ballock": "pollock",
+		"fist": "fish",
+		"place": "plaice",
+		"definitively" : "dolphinitely",
+		"enemy": "anemony",
+		"enemies": "anemones",
+		" mob": " lob",
+		"bitch": "beach",
+		" min ": "minnow",
+		"lemme know": "lemminnow",
+		"let me know": "let minnow",
+		"admin": "adminnow",
+		"better": "betta",
+		" ass ": " wrasse ",
+		"asshole" : "wrassehole",
+		"bond" : "pond",
+		"inc": "ink",
+		"anything": "anyfin",
+		"something": "somefin",
+		"anythin": "anyfin",
+		"somethin": "somefin",
+		"someone": "salmon",
+		"coy": "koi",
+		"earring": "herring",
+		"hearin": "herrin",
+		"celebrat": "shellebrat",
+		"santa": "sandy",
+		"claus": "claws",
+		"boss": "bass",
+		"doofus": "doofish",
+		"could": "cod",
+		"what are": "water",
+		"what're": "water",
+		"whatre": "wa'er",
+		"other": "otter",
+		"lots": "lobst",
+		"fumble": "flounder",
+		"serious": "searious",
+		"show": "shoal",
+		"miracle": "mackerel",
+		"trait": "trout",
+		"syndicate": "sea world",
+		"syndies": "fishermen",
+		"syndie": "fisherman",
+		"syndi": "fisherman",
+		"spy": "marine biologist",
+		"above": "upstream",
+		"below": "downstream",
+		"heretic": "herringtic",
+		"cult": "shoalt",
+		"nar'si": "carp'si",
+		"narsi": "carpsi",
+		"blob": "blobfish",
+		"dumbass": "dum bass",
+		"outdone": "troutdone",
+		"quintessen": "squiddessen",
+		"clos": "claws",
```

Comment:
Is this meant to be "close"? I don't know what "clos" is

Reply:
Yes, it should fit on words like close, closet, closing and closure: clawse, clawset, clawsing, clawsure.